pkg/splunk/enterprise/util.go

```diff
@@ -1682,19 +1448,17 @@ func CopyFileToPod(c splcommon.ControllerClient, namespace string, podName strin
 		return "", "", fmt.Errorf("relative paths are not supported for dest path: %s", destPath)
 	}
 
-	// Make sure the destination directory is existing
```

Comment:
How was the directory getting created before this change?

Reply:
init container was responsible for creating the directory and downloading the app/s to that directory